### DISCUSSION TITLE
Creation of snupkg package

### DISCUSSION THREAD

**Discussion by donbabbeo** [devlooped/nugetizer#530] [Q&A] [answered]
Hello,

thanks for the tool, it make dealing with references a breeze, but i'm trying to figuring out how to create a separate .snupkg for debug symbol and i'm somewhat stumped.

As stated [here](https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg) it's enough to add `IncludeSymbols` and `SymbolPackageFormat` to the csproj.
When using nugetizer the latter seems ignored and the former simply means 'add .pdb to the nupkg along side the dlls'.

There is some way to replicate the function of dotnet pack and create a snupkg alongside the nupkg?

If may add another question: the solution that want to packetize contains some old csproj projects (i'm targeting net48) and apparently there is no way to exclude those pdb from the output, oldcsproj seems lacking the `IncludeSymbols` option. I know that this tools is not meant to be used with old csproj but there is some workaround?

**Comment by kzu** [comment, marked as answer]
Hi there! I don't believe in separate symbol packages ;-). The portable pdb format is extremely compact, and can even be embedded in the dlls. Separate symbol packages add little to no value IMHO. Hence, they are not supported in nugetizer. 

I haven't tested things with old csproj at all, but I heard the new conversion tool to SDK-style should make that a breeze. Perhaps you can update the project file, while keeping net48 as the target?

**Reply by donbabbeo** [reply]
Thanks for the response! 
I understand your reasoning, it's a matter of preference more than usefullness honestly, i like keeping stuffs separated so i would have preferred some sort of flag to deal with this. 
No big deal, i can always make it with some sort of script.

Sadly the csproj switch it's not something that's under my control, i've tested the conversion to SDK-style and It works flawlessly but for the time being is out of the picture.
